pyvex/lift/util/syntax_wrapper.py

```diff
@@ -124,31 +126,59 @@ def __add__(self, right):
         return self.irsb_c.op_add(self.rdt, right.rdt)
 
     @checkparams
+    def __radd__(self, left):
+        return self + left
+
+    @checkparams
     @vvifyresults
-    def __sub__(self, right): # I'm not sure if we want rsub
+    def __sub__(self, right):
         return self.irsb_c.op_sub(self.rdt, right.rdt)
-
+    """
+    @checkparams
+    def __rsub__(self, left):
+        # TODO: FIXME
+        return left - self
+    """
     @checkparams
     @vvifyresults
     def __div__(self, right):
-        return self.irsb_c.op_div(self.rdt, right.rdt, self.is_signed)
-
+        if self._is_signed:
+            return self.irsb_c.op_sdiv(self.rdt, right.rdt)
+        else:
+            return self.irsb_c.op_udiv(self.rdt, right.rdt)
+    """
+    @checkparams
+    def __rdiv__(self, left):
+        return left / self
+    """
     @checkparams
     @vvifyresults
     def __floordiv__(self, right): # Note: nonprimitive
         return self / right
-
+    """
+    @checkparams
+    def __rfloordiv__(self, left):
+        return left // self
+    """
     @checkparams
     @vvifyresults
     def __truediv__(self, right): # Note: nonprimitive
         return self / right
-
+    """
+    @checkparams
+    def __rtruediv__(self, left):
+        return left.__truediv__(self)
+    """
```

Comment:
generally we don't like using strings for multiline comments. just stick a # in front of them. If you're using a good editor this should be easy.